src/components/views/dialogs/ForwardDialog.tsx

```diff
@@ -141,8 +141,10 @@ const Entry: React.FC<IEntryProps> = ({ room, type, content, matrixClient: cli,
         >
             <DecoratedRoomAvatar room={room} avatarSize={32} />
             <span className="mx_ForwardList_entry_name">{ room.name }</span>
-            { detailsText && <span className="mx_ForwardList_entry_detail">
-                { detailsText }
+            { contextDetails && <span
+                className="mx_ForwardList_entry_detail"
+                aria-label={contextDetails.ariaLabel}>
+                { contextDetails.details }
             </span> }
```

Comment:
Can we not reuse RoomResultDetails here? (by making the className a prop and moving it to a shared dir)